src/api.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://board-games-api-q1cq.onrender.com/api/",
+});
+
+export const fetchReviews = () => {
+  return api.get("/reviews").then((response) => {
+    return response.data.reviews;
```

Comment:
Awesome to see you just return the key you need here rarther than the full object 🙌